packages/code/src/types.ts

```diff
@@ -118,16 +111,23 @@ export type CodeProps = Omit<
   onCopy?: Function;
 
   /**
-   * Custom action buttons.
+   * Custom action buttons. Should be an array of `IconButton`.
    *
    */
-  customActionButtons?: Array<React.ReactNode>;
+  customActionButtons?: Array<React.ReactElement>;
```

Comment:
Why `ReactElement` over `ReactNode`?

Reply:
Also we should add a custom TSDoc annotation here:
`@type <IconButton />[]`

Reply:
I was thinking that `ReactElement` was a little stricter. We already have a check to make sure that the customActionButtons are `IconButton` so I didn't see the harm in changing it.

Reply:
Oh, yeah, that makes sense 👍